tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -555,10 +558,13 @@ describe('Creating And Updating Ranked Matches', () => {
     cy.window()
       .its('cuttle.app.config.globalProperties.$store.state.game')
       .then((game) => {
-        const results = game.currentMatch.games.map((g) => g.result);
-        cy.expect(results[0]).to.eq(0);
-        cy.expect(results[1]).to.eq(0);
-        cy.expect(results[2]).to.eq(2);
+        const results = game.currentMatch.games.map(({status, winner}) => ({status, winner}));
+        cy.expect(results[0].status).to.eq(3);
+        cy.expect(results[0].winner).to.eq(this.playerOneId);
+        cy.expect(results[1].status).to.eq(3);
+        cy.expect(results[1].winner).to.eq(this.playerTwoId);
+        cy.expect(results[2].status).to.eq(3);
+        cy.expect(results[2].winner).to.eq(null);
```

Comment:
I think there is something funky here `cy.expect(results[1]).to.eq(0);` should mean p0 won the this game, but `   cy.expect(results[1].winner).to.eq(this.playerTwoId);` passes